content/guides/component/src/routes.clj

```diff
@@ -0,0 +1,13 @@
+ ;; tag::ns[]
+(ns routes)
+ ;; end::ns[]
+
+ ;; tag::response[]
+(defn respond-hello [request]
+  {:status 200 :body "Hello, world!"})
+ ;; end::response[]
+
+ ;; tag::routes[]
+(def routes
```

Comment:
I'd prefer to leave this to the next component guide I referenced earlier. That's where I'll be demonstrating some of the additional plumbing the Pedestal component should implement to support needs such as the one you mentioned.